dev/ubuntu20.04.Dockerfile

```diff
@@ -1,10 +1,22 @@
+#
+# Note: If building this container from a machine running non-X86 hardware
+# (like Apple M1 / M2 chips), you'll need to build and run the container
+# specifying the platform e.g.
+# $ docker build --platform linux/amd64 ubuntu20.04.Dockerfile
+#
 FROM ubuntu:20.04
 WORKDIR /home/
 
-RUN ["apt-get", "-y", "upgrade"]
-RUN ["apt-get", "-y", "update"]
-RUN ["apt-get", "install", "-y", "git"]
+ENV LANG="C.UTF-8" LC_ALL="C.UTF-8"
+
+RUN apt-get -y upgrade \
+    && apt-get -y update \
+    && apt-get install -y \
+        build-essential \
+        git
 RUN ["git", "clone", "https://github.com/GaloisInc/cryptol.git"]
 
-COPY dev_setup.sh ./cryptol/
-CMD ["/bin/bash"]
\ No newline at end of file
+WORKDIR /home/cryptol
+RUN ["dev/dev_setup.sh"]
+RUN ["source", "dev/env.sh"]
```

Comment:
This will, unfortunately, probably run a shell, load the environment settings into it, then exit and throw them away. I suspect we need to create a .profile or similar in the container that sources env.sh...

Reply:
Oops, thanks gh. Maybe you can tell me that I'm making an outdated comment _before_ posting it? :-(

N/M.

Reply:
Yeah. Actually it couldn't even run the source line, I kept getting errors like `"source": executable file not found in $PATH` and `source /home/cryptol/dev/env.sh: No such file or directory` despite trying to manually create it in the line before. I tried sourcing it with `. dev/env.sh` because I guess the default shell is dash and doesn't have source, but that didn't work, and I tried switching to a bash shell and then running the command, and running the command with normal `RUN source dev/env.sh` notation instead of the list like here, but it absolutely refused to ever find all the parts it needed. Hence the giving up and setting env vars by hand.